content/definitions_to_custom_resources.md

```diff
@@ -1,20 +1,24 @@
 +++
 title = "Converting Definitions to Custom Resources"
-draft = false
-
 gh_repo = "chef-web-docs"
 
-aliases = ["/definitions.html"]
+product = ["client", "workstation"]
 
 [menu]
   [menu.infra]
-    title = "Converting Definitions to Custom Resources"
-    identifier = "chef_infra/resources/definitions.md Migrating from Definitions"
-    parent = "chef_infra/resources"
-    weight = 20
+    title = "Migrating from Definitions"
+    identifier = "chef_infra/resources/custom_resources/definitions.md Migrating from Definitions"
+    parent = "chef_infra/resources/custom_resources"
+    weight = 110
 +++
 
-This topic covers migrating existing definitions to custom resources. Custom resources are integral to the modern Chef Infra workflow. While definitions are not yet deprecated, we *strongly* advise migrating to custom resources immediately. You'll benefit from the many features of Chef Infra such as notifications, reporting, why-run mode, and ChefSpec unit testing.
+## Migrating from Definitions to Custom Resources
+
+The definitions feature in Chef Infra has been deprecated and will be removed in a future release. Please migrate existing definitions to Custom Resources.
```

Comment:
We have not yet committed to removing the feature. We're just strongly advocating for folks getting off them.